packages/cfd/src/Constants/jurisdiction-contents.ts

```diff
@@ -1,12 +1,63 @@
 import { localize } from '@deriv/translations';
+import {
```

Comment:
I suggest breaking this file down into multiple config files because it is too large